tests/wpt/mozilla/tests/mozilla/microdata/json.html

```diff
@@ -0,0 +1,12 @@
+<!doctype html>
+<meta charset="utf-8">
+<title>JSON</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<script>
+  test(function () {
+      /* do something long and slow here */
```

Comment:
Not sure what the long and slow means; if you want automated tests for this code, I recommend adding a test-only APIs to Document.webidl that return the result of extracting the microdata as a string that you can compare against expected results.

Reply:
We will remove this test case. For now we plan to use a unix diff command to compare the expected and generated output.